<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtlBrowserDialog</class>
 <widget class="QtlDialog" name="QtlBrowserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonBackward">
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="buttonClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="6">
    <widget class="QTextBrowser" name="text">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="buttonForward">
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="buttonHome">
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtlDialog</class>
   <extends>QDialog</extends>
   <header>QtlDialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>text</sender>
   <signal>backwardAvailable(bool)</signal>
   <receiver>buttonBackward</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>text</sender>
   <signal>forwardAvailable(bool)</signal>
   <receiver>buttonForward</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBackward</sender>
   <signal>clicked()</signal>
   <receiver>text</receiver>
   <slot>backward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonForward</sender>
   <signal>clicked()</signal>
   <receiver>text</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonHome</sender>
   <signal>clicked()</signal>
   <receiver>text</receiver>
   <slot>home()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>QtlBrowserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
